samples/nrf9160/modem_shell/Kconfig

```
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Modem Shell config"
+
+config MOSH_IPERF3
+	bool  "Enable iperf3 support"
+	default y
+	select NRF_IPERF3
+
+config MOSH_SOCK
+	bool  "Enable socket tool support"
```

The ability to select / deselect implies that the user is enabling something, see also the guide lines:
https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#prompt-strings
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like. It will usually be clear in the context of an option that can be toggled on/off, and makes things consistent.

```suggestion
	bool  "Socket tool support"
```

Also missing help text.
What kind of socket tool support, tcp, udp, raw, other, or all of them ?

---

Help text added. Documentation also has information about some of the Modem Shell features.

---

> Help text added.

Great, thanks :+1:

> Documentation also has information about some of the Modem Shell features.

I know, but for a user who is exploring the sample or using menuconfig then it's nice to have this info readily available.